src/lib/cssCustomProperties.js

```diff
@@ -0,0 +1,6 @@
+// This is needed to generate CSS with definitions to be used in docs directly (not in shadow DOM)
+/* eslint-disable no-unused-vars */
```

Comment:
What is the reason have imports commented out? If we do not need it, removed it together with eslint-disable.

Reply:
fixed